### PR TITLE
Add hostname to /v1/

### PR DESCRIPTION
**Pull request by raphink**
Signed-off-by: Raphaël Pinson <raphael@isovalent.com>

### CODE REVIEW

**Review by nebril — APPROVED** [review]

File: restapi/configure_deathstar.go [hunk 1]
```diff
@@ -41,14 +46,11 @@ var info = `{
 	"starship_class": "Deep Space Mobile Battlestation",
 	"api": [
 		"GET   /v1",
-		"GET   /v1/healthz",
 		"POST  /v1/request-landing",
-		"PUT   /v1/cargobay",
-		"GET   /v1/hyper-matter-reactor/status",
```
Comment (aanm) [inline restapi/configure_deathstar.go]: why were these deleted?

**Review by aanm — CHANGES_REQUESTED** [review]

File: restapi/configure_deathstar.go (hunk 1 above)
Comment (raphink) [reply restapi/configure_deathstar.go]: It seems these endpoints were never implemented, so if we're going to use this `/v1/` info interface (which we didn't so far), it's probably better to remove them.

I'm happy to make a separate PR for it if you prefer though.

Comment (aanm) [reply restapi/configure_deathstar.go]: This `info` is used for any GETs performed to the server. We should keep them since it looks good on demos even if they are not implemented.

Comment (raphink) [reply restapi/configure_deathstar.go]: OK then, I'll keep them!

**Review by aanm — APPROVED** [review]
